package.json

```diff
@@ -14,6 +14,7 @@
     "test:lint": "standard --env mocha",
     "test:unit": "nyc tape 'src/web/**/*.test.js'",
     "test:acceptance": "mocha 'src/test/acceptance/**/*.js'",
+    "test:cucumber": "./node_modules/.bin/cucumber-js src/test/acceptance/features/",
```

Comment:
I think that `./node_modules/.bin/cucumber-js` could be replaced with `cucumber`